packages/layout/src/components/cards/common/views/remove/date/date.tsx

```diff
@@ -38,18 +37,19 @@ const DateForm: React.FC<DateFormProps> = ({
 			>
 				{({ handleSubmit, submitError }) => (
 					<form onSubmit={handleSubmit} data-testid="remove-actuary-form">
-						<FFCheckbox
-							name="confirm"
-							type="checkbox"
-							label={label}
-							cfg={{ mb: 3 }}
-						/>
-						<div className={styles.dateWrapper}>{renderFields(dateField)}</div>
+						<fieldset aria-describedby="error-msg">
```

Comment:
Does this render a nested fieldset? For accessibility reasons, this isn't recommended. https://accessibility.blog.gov.uk/2016/07/22/using-the-fieldset-and-legend-elements/

Reply:
@dotcom9  what element would you suggest we use to wrap the `checkbox` and the `date field` for linking it to the error message passed to the `aria-describedby` attribute?